resources/type-system/upper-lower-bounds.md

```diff
@@ -280,6 +294,16 @@ follows.
 
 - **DOWN**(`T Function<...>(...)`, `S Function<...>(...)`) = `Never` otherwise
 
+- **DOWN**(`(...)`, `Record`) = `(...)`
+- **DOWN**(`Record`, `(...)`) = `(...)`
```

Comment:
It looks that we don't have similar statements for function types, at least in the analyzer. And I don't see them here. Instead it works because we use `DOWN(T1, T2) = T1 if T1 <: T2` and `DOWN(T1, T2) = T2 if T2 <: T1`.

Reply:
Yeah, that's cleaner.  Fixed.